src/transformers/models/megatron_gpt2/checkpoint_reshaping_and_interoperability.py

```diff
@@ -607,7 +607,14 @@ def convert_checkpoint_from_transformers_to_megatron(args):
         sys.path.insert(0, args.megatron_path)
 
     try:
-        from megatron.tokenizer.tokenizer import _vocab_size_with_padding
+        from megatron.core import package_info
+
+        version = package_info.__version__
+        version = tuple(map(int, version.split(".")))
+        if version < (0, 6):
+            from megatron.tokenizer.tokenizer import _vocab_size_with_padding
+        else:
+            from megatron.training.tokenizer.tokenizer import _vocab_size_with_padding
     except ModuleNotFoundError:
```

Comment:
Thanks, you can use the utils: https://github.com/huggingface/transformers/blob/main/src/transformers/utils/import_utils.py#L42

Reply:
@ArthurZucker 

Megatron-LM is not a distributed package, so I couldn't use "https://github.com/huggingface/transformers/blob/main/src/transformers/utils/import_utils.py#L42".
 Therefore, I modified the code to use importlib.util to check the importability of Megatron-LM.